plugins/module_utils/compose_v2.py

```diff
@@ -701,23 +701,30 @@ def __init__(self, client, min_version=MINIMUM_COMPOSE_VERSION):
         self.env_files = parameters['env_files']
         self.profiles = parameters['profiles']
 
-        compose = self.client.get_client_plugin_info('compose')
-        if compose is None:
-            self.fail('Docker CLI {0} does not have the compose plugin installed'.format(self.client.get_cli()))
-        if compose['Version'] == 'dev':
-            self.fail(
-                'Docker CLI {0} has a compose plugin installed, but it reports version "dev".'
-                ' Please use a version of the plugin that returns a proper version.'
-                .format(self.client.get_cli())
-            )
-        compose_version = compose['Version'].lstrip('v')
-        self.compose_version = LooseVersion(compose_version)
-        if self.compose_version < LooseVersion(min_version):
-            self.fail('Docker CLI {cli} has the compose plugin with version {version}; need version {min_version} or later'.format(
-                cli=self.client.get_cli(),
-                version=compose_version,
-                min_version=min_version,
-            ))
+        # Check "docker compose version" first as this allows for "podman compose" to work as well
+        rc, version, stderr = self.client.call_cli('compose', 'version', '--short')
```

Comment:
I think it would be better to use the JSON output and parse that.
```suggestion
        rc, version, stderr = self.client.call_cli('compose', 'version', '--format', 'json')
```